### PR TITLE
Resources

### LINKED ISSUE #5: User can View All Resources
As a user, when I visit my dashboard, I want to click the "All Resources" button, so I can view all resources and a section for additional resources. 

- [x] Add embedded video player with the following playlists: yoga, meditation, and motivational
- [x] Add aside for Additional Resources with: suicide prevention hotline, and mental health/resources links

### PR DESCRIPTION
**Pull request by danielbldr**
## What functionality does this accomplish?
- Created all resources view
- Added embedded video player in all resources view
- Added links to render yoga, meditation, and motivational videos 
- Created VideoService for API calls to microservice. 
- Mocked VideoService get_videos method with expected JSON response until microservice is built
- Created Video poro to pass video objects in the view
- Created VideoResults poro to create video objects from JSON response
- Created PlaylistFacade to store videos for specific category and return current video

closes #5
**Description:**
Created All Resources view to return all yoga, motivational, and meditation videos. The page has an embedded youtube player and see all videos within each playlist. There is also a section for external resources. 

### Test Coverage Percentage: 98.96% --- Can't test VideoService fully until microservice is created. Instead, we stubbed out the response to the get_videos method
- [x] No tests have been changed
- [ ] Some tests have been changed
- [ ] All of the tests have been changed(Please describe what in the world happened):
### Rubocop:
- [x] No offenses
- [ ] Offenses (number of offenses: _ )
## Checklist:
- [ ] My code has no unused/commented out code --- One line of commented out code so we know what to add to the get_videos method once the microservice is created
- [x] I have reviewed my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have fully tested my code
- [x] I have partially tested my code (please explain why):
per the reasons above
## Helpful Resources:
* Resource link AND small description of info gathered/learned

### CODE REVIEW

File: app/views/all_resources/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<section class='row'>
```
Comment (TravisBorgsmiller) [inline app/views/all_resources/index.html.erb:1]: This JS looks really good

File: app/poros/video_results.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class VideoResults
+  def create_videos(category)
+    video_service = VideoService.new
+    videos = video_service.get_videos(category)
+    videos[:data].map { |yoga_vid| Video.new(yoga_vid) }
```
Comment (javier-aguilar) [inline app/poros/video_results.rb:5]: The naming of yoga_vid leads me to assume all the videos are yoga related? Is that the case?

File: app/services/video_service.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  def get_videos(category)
+    # get_json('/api/v1/videosfind_all', category)
+    response = if category == 'motivational'
+                 File.read('app/mock/motivational_video.json')
+               elsif category == 'meditation'
+                 File.read('app/mock/meditation_video.json')
+               else
+                 File.read('app/mock/yoga_video.json')
+               end
+    JSON.parse(response, symbolize_names: true)
```
Comment (javier-aguilar) [inline app/services/video_service.rb:11]: If you wanted to refactor, you could do 
`File.read("app/mock/#{category}_video.json")`

File: app/poros/video_results.rb (hunk 2 above)
Comment (SMJ289) [reply app/poros/video_results.rb:5]: I think that is the category we started with so we should rename it to just videos for clarity.

File: app/services/video_service.rb (hunk 3 above)
Comment (SMJ289) [reply app/services/video_service.rb:11]: We could but this code is just temporary before we build out the microservice.

**Pull request merged**